242.valid-anagram.js

```diff
@@ -0,0 +1,26 @@
+/* URL of this problem
+    https://leetcode.com/problems/valid-anagram/
+*/
+
+/*
+ * @param {string} s
+ * @param {string} t
+ * @return {boolean}
+ */
+ 
+var isAnagram = function(s, t) {
+    const sArr = [...s].sort();
+    const tArr = [...t].sort();
+    
+    if (sArr.length !== tArr.length) {
+        return false;
+    }
+    
+    for (let i = 0; i < sArr.length; i++) {
+        if (sArr[i] !== tArr[i]) {
+            return false;
+        }
+    }
+    
+    return true;
```

Comment:
```suggestion
    return sArr.join("") === tArr.join("");
```
You can also write like this.

Reply:
@yuya-minamide it looks good

Reply:
I modified the code as the suggested change above.